### DISCUSSION TITLE
Feature Questions

### DISCUSSION THREAD

**Discussion by prsnetwork** [bluenviron/mediamtx#574] [General]
Hi first thank you for developing this finding it really useful.

Does it support more then one readuser
Could do with multiple user names and passwords against a proxy stream.

Is it possible for it to run as a service on windows using sc.exe currently have it working with a third party service manager but would be great if it used built in windows service manager. I am aware it’s works as service on Linux.

Thanks again

**Comment by aler9** [comment]
> Does it support more then one readuser

The server will support external authentication in the future, please follow this #517

> Is it possible for it to run as a service on windows using sc.exe 

Use nsmm to wrap the executable inside a Window service:

https://nssm.cc/

[a comment by an automated account is not shown]
